payjoin/src/receive/mod.rs

```diff
@@ -359,10 +360,18 @@ impl UncheckedProposal {
     /// Broadcasting the Original PSBT after some time in the failure case makes incurs sender cost and prevents probing.
     ///
     /// Call this after checking downstream.
-    pub fn check_can_broadcast(
+    pub fn check_broadcast_suitability(
```

Comment:
we probably want to update the documentation of the function to reflect the current change